packages/test/test-end-to-end-tests/src/test/createContainerStats.spec.ts

```diff
@@ -25,9 +24,13 @@ class TestDataObject extends DataObject {
     public get _root() {
         return this.root;
     }
+
+    public get containerRuntime() {
+        return this.context.containerRuntime;
+    }
 }
 
-describeNoCompat("Generate Summary Stats", (getTestObjectProvider) => {
+describeNoCompat.only("Generate Summary Stats", (getTestObjectProvider) => {
```

Comment:
`.only` here, remove?

Reply:
Good catch! This would have been disastrous :D